src/assets/css/style.scss

```diff
@@ -1,4 +1,12 @@
 ---
+# Keep this YAML front matter block to trigger Jekyll processing
```

Comment:
I found the requirement of the YAML blocks as frontmatter confusing, so copied a note in—feel free to remove